cf-agent/audit_mode.c

```diff
@@ -342,7 +345,104 @@ bool ManifestRename(const char *orig_name, const char *new_name)
     return true;
 }
 
-bool ManifestChangedFiles()
+static bool RunDiff(const char *path1, const char *path2)
+{
+    char diff_path[PATH_MAX];
+    strncpy(diff_path, GetBinDir(), sizeof(diff_path));
+    JoinPaths(diff_path, sizeof(diff_path), "diff");
+    char *command;
+    int ret = xasprintf(&command, "%s -u --label 'original %s' --label 'changed  %s' '%s' '%s'",
+                        diff_path, path1, path1, path1, path2);
```

Comment:
Should it be `1, 1, 1, 2` or `1, 2, 1, 2`? If it is correct how you wrote it, that both labels should have `path1`, then I think this deserves a comment.

Reply:
It is correct because the output then looks like this:
```
--- original /etc/motd.d/cfengine
+++ changed  /etc/motd.d/cfengine
```
I'll add an example output. :+1: